tests/unit/model/test_container.py

```diff
@@ -0,0 +1,91 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Ensure the expected behaviour of the container element."""
+
+
+import pytest
+
+from structurizr.model import Component, Container, Model
+
+
+_model = (
+    Model()
+)  # Have to create outside the fixture so it doesn't get garbage-collected.
```

Comment:
I'm going to look at whether this hack can be avoided by instead mocking the SoftwareSystem of the Container.